### PR TITLE
Update the README file

### PR DESCRIPTION
**Pull request by roytman**
- add recommendation to init helm ( `helm init --client-only` ) before packaging charts in the local storage
- fix location of the `helm repo add submariner-latest` setting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,9 +93,9 @@ Search the new repo for submariner charts:
 helm search -l test-repo
 ```
 
-### Modify submariner e2e.sh script to use your local test-repo.
+### Modify submariner e2e tests helm deployment script to use your local test-repo.
 
-Change the line from:
+In the file `scripts/kind-e2e/lib_helm_deploy_subm.sh` change the line from:
```
Comment (dfarrell07) [inline README.md:98]: I think this is now in Shipyard's `scripts/shared/lib/deploy_helm`.

Comment (roytman) [reply README.md]: Wow, everything is changing so fast. 
Fixed the filename and added a link to the repository.

File: README.md [hunk 2]
```diff
@@ -87,9 +93,10 @@ Search the new repo for submariner charts:
 helm search -l test-repo
 ```
 
-### Modify submariner e2e.sh script to use your local test-repo.
+### Modify submariner e2e tests helm deployment script to use your local test-repo.
 
-Change the line from:
+You can test your helm-charts with e2e tests from the [shipyard](https://github.com/submariner-io/shipyard) repository.
+In the file `iscripts/shared/lib/deploy_helm` change the line from:
```
Comment (dfarrell07) [inline README.md:99]: Little typo here (Extra "i" at the start of the file path)

File: README.md (hunk 1 above)
Comment (dfarrell07) [reply README.md]: Yeah, lots going on, lots getting better. :) Sorry to be a bit slow and let this rot under you.

File: README.md (hunk 2 above)
Comment (roytman) [reply README.md]: Thanks for pointing me. Fixed.

**Pull request merged**